webknossos/webknossos/skeleton/group.py

```diff
@@ -20,10 +20,15 @@
 class Group:
     _id: int = attr.ib(init=False)
     name: str
-    _children: Set[GroupOrGraph] = attr.ib(
+    _child_groups: Set["Group"] = attr.ib(
         factory=set,
         init=False,
-        repr=lambda children: f"<{unit_len(children, 'children')}>",
+        repr=lambda children: f"<{unit_len(children, 'child groups')}>",
```

Comment:
```python
>>> unit_len([1, 2], 'child groups')
'2 child groupses'
```
:rofl:

```suggestion
        repr=lambda children: f"<{unit_len(children, 'child group')}>",
```

Reply:
:sweat_smile: 

I thought, one would need to provide the plural form. However, I now see, that this doesn't make much sense. However, I just wanted to say that `unit_len` also isn't perfect (`unit_len(["one"], 'children')` yields `1 children` even though I would have expected `1 child`).

Reply:
> However, I just wanted to say that `unit_len` also isn't perfect (`unit_len(["one"], 'children')` yields `1 children` even though I would have expected `1 child`).

It is, if you use the singular form here as well:
```python
>>> unit_len([1, 2], 'child')
'2 children'
>>> unit_len([1], 'child')
'1 child'
```

Reply:
Ah ok, then it has been wrong before this PR ;)